client/state/initial-state.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import debugModule from 'debug';
+import localforage from 'localforage';
+
+/**
+ * Internal dependencies
+ */
+import { createReduxStore } from 'state';
+import { reducer } from 'state';
+import { TO_OBJECT, FROM_OBJECT } from 'state/action-types'
+
+/**
+ * Module variables
+ */
+const debug = debugModule( 'calypso:state' );
+
+const localforageConfig = {
+	name: 'calypso',
+	storeName: 'calypso_store',
+	description: 'Calypso Storage'
+};
+
+function toObject( state ) {
+	return reducer( state, { type: TO_OBJECT } );
```

Comment:
Naming-wise, could I suggest we use naming standards proposed in aff948ec88f67dd58ddc052db75ac5199355488b (suffixing verb)? Something like `INITIAL_STATE_RESTORE` and `INITIAL_STATE_STORE` ? That said, I do like the simplicity and clarity of what you've chosen here.